src/richie/apps/courses/urls.py

```diff
@@ -14,4 +15,9 @@
 
 urlpatterns = ROUTER.urls + [
     re_path("course-runs-sync/?$", course_runs_sync, name="course_run_sync"),
+    path(
+        "person-page-autocomplete/",
+        PersonPageAutocomplete.as_view(),
+        name="person-page-autocomplete",
```

Comment:
We need to differentiate it from Elasticsearch autocomplete for clarity
```suggestion
        name="person-page-admin-autocomplete",
```